corehq/apps/cleanup/management/commands/fix_corrupted_forms.py

```diff
@@ -25,25 +16,42 @@ def add_arguments(self, parser):
         parser.add_argument('domain')
         parser.add_argument('app_id')
         parser.add_argument('form_id')
-        parser.add_argument(
-            '--cleanup',
-            action='store_true',
-            dest='cleanup',
-            default=False,
-        )
 
     # https://dimagi-dev.atlassian.net/browse/HI-747
-    def handle(self, domain, app_id, form_id, cleanup=False, **options):
-        app = get_apps_by_id(domain, app_id)[0]
+    def handle(self, domain, app_id, form_id, **options):
+        app = get_app(domain, app_id)
         form = app.get_form(form_id)
-        source = form.source
-        if any(suspicious_string in source for suspicious_string in SUSPICIOUS_STRINGS):
-            print('FORM CONTAINS SUSPICIOUS STRING')
-            if cleanup:
-                if 'y' == input('Did you confirm that there are no app updates to publish? [y/N]'):
-                    print('Cleaning form...')
-                    form.source = source.encode('latin1').decode('utf-8')
-                    app.save()
-                    print('Done.')
-                else:
-                    print('Aborting...')
+        old_source = form.source
+        new_source = fix_form(old_source)
+        if old_source != new_source:
+            if input("commit the above changes?\n[y/N] ") == 'y':
+                form.source = new_source
+                app.save()
+                print("saved")
+            else:
+                print("aborting")
+        else:
+            print("There was no change")
+
+
+def latin_to_utf(latin_string):
```

Comment:
Nit: It would make it easier for the next person if a docstring explained that `latin_string` is Unicode, and it contains characters decoded from `latin1` that should have been decoded from `utf-8`.

Reply:
committed @millerdev's proposed docstring